src/test/java/org/squbs/sample/SampleHttpServiceTest.java

```diff
@@ -0,0 +1,100 @@
+package org.squbs.sample;
+
+import akka.http.javadsl.Http;
+import akka.http.javadsl.model.*;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import org.junit.After;
+import org.junit.Test;
+import org.squbs.testkit.japi.AbstractCustomTestKit;
+
+import static org.junit.Assert.assertEquals;
+
+public class SampleHttpServiceTest extends AbstractCustomTestKit {
```

Comment:
Based on my current changes in https://github.com/paypal/squbs/pull/439, this `AbstractCustomTestKit` needs to be changed to `CustomTestKit`.